### PR TITLE
#272 Add Mssql, Oracle, Cubrid, Postgresql, Cassandra, Mongodb, RabbitMq, …

### PR DESCRIPTION
**Pull request by marty-macfly**
…ActiveMq, Arcus

### CODE REVIEW

**Comment by marty-macfly** [comment]
Link to issue #272

File: plugins/PHP/Plugins/Common/PluginsDefines.php [hunk 1]
```diff
@@ -38,14 +38,23 @@
...
 define("PP_REDIS_REDISSON","8203");
 define("PP_REDIS_REDISSON_INTERNAL","8204");
+define("PP_RABBITMQ", "8300");
+define("PP_ACTIVEMQ", "8310");
+define("PP_ARCUS", "8100");
 define("PP_MEMCACHED","8050");
 define("PP_PHP_REMOTE",9700);
 
-define("PP_HEADER_NGINX_PROXY","Pinpoint-ProxyNginx");
+define("PP_HEADER_NGINX_PROXY","HTTP_PINPOINT_PROXYNGINX");
```
Comment (eeliu) [inline plugins/PHP/Plugins/Common/PluginsDefines.php:57]: Pinpoint-ProxyNginx can't change to HTTP_PINPOINT_PROXYNGINX. 
Nginx will keep the case, while apache not.

Comment (marty-macfly) [reply plugins/PHP/Plugins/Common/PluginsDefines.php]: Ok, I've revert the change.

**Review by eeliu — APPROVED** [review]
LGTM

**Pull request merged**